src/doc/src/reference/specifying-dependencies.md

```diff
@@ -454,8 +454,51 @@ following to the above manifest:
 log-debug = ['bar/log-debug'] # using 'foo/log-debug' would be an error!
 ```
 
+### Inheriting a dependency from a workspace
+
+Dependencies can be inherited from a workspace by specifying the
+dependency in the workspace's [`[workspace.dependencies]`][workspace.dependencies] table.
+After that add it to the `[dependencies]` table with `dep.workspace = true`.
```

Comment:
Here `dep` may not be clear that this is a placeholder for what the user actually enters. Perhaps just not mention it? I think the example below is what most users will actually look at, which should make it clear what the syntax looks like. Maybe like this:

```suggestion
After that add it to the `[dependencies]` table with `workspace = true`.
```

Reply:
I used what you put here. I was trying to say "After that add it to the `[dependencies]` table with `{dependency name}.workspace = true`."